components/cycles/CreateCycle.jsx

```diff
@@ -32,28 +38,53 @@ const AddCycleButton = styled(PlusButton)`
    bottom: 20px;
    right: 25px;
 `;
+
 export default ({ navigation }) => {
   const [name, setName] = React.useState('');
+  const [workouts, setWorkouts] = React.useState(workoutsSampleData);
 
+  const Stack = createStackNavigator();
   return (
-    <View style={{ height: '100%' }}>
-      <View>
-        <BackButton onPress={() => navigation.navigate('Cycles')}>
-          <AntDesign name="left" size={30} color="black" />
-        </BackButton>
-        <TitleTextInput
-          style={{ borderColor: name ? 'transparent' : 'black' }}
-          onChangeText={(newName) => setName(newName)}
-          value={name}
-          placeholder="Cycle Name"
-        />
 
-      </View>
+    <SafeAreaView style={{ height: '100%' }}>
+
+      <BackButton onPress={() => navigation.navigate('Cycles')}>
+        <AntDesign name="left" size={30} color="black" />
+      </BackButton>
+
+      <TitleTextInput
+        style={{ borderColor: name ? 'transparent' : 'black' }}
+        onChangeText={(newName) => setName(newName)}
+        value={name}
+        placeholder="Cycle Name"
+      />
+
       <AddFinishButton onPress={() => alert('Cycle Created')} />
-      {/* Finish Button will take u back to Cycles and add cycle to list */}
-      {/* This is where a list of workouts in the cycle will go. Should be
-      able to drag to change order */}
-      <AddCycleButton title="Workout" size={18} onPress={() => alert('Add Workout to Cycle')} />
-    </View>
+
```

Comment:
Should we add finish button functionality before the presentation?

Reply:
Yeah I'll add that w redux, redux will also let us display the workout we create in allCycles view